trunk/SUAVE/Input_Output/Results/plot_constraint_diagram.py

```diff
@@ -0,0 +1,103 @@
+## @ingroup Input_Output-Results
```

Comment:
this should be moved to this folder: https://github.com/suavecode/SUAVE/tree/develop/trunk/SUAVE/Plots/Performance